bot.py

```diff
@@ -39,6 +67,8 @@ async def on_command_error(ctx, error):
 			raise e
 
 with open('config.config', 'r') as f:
-	tok = f.readline()
-	tok.replace('\n', "")
-bot.run(tok)
+	for i in f:
+		splitted = i.split(':')
+		if splitted[0].lower() == "token":
+			token = splitted[1]
+bot.run(token)
```

Comment:
Rather than opening and parsing this twice, can it just be done once? If it could be done in a more atomic way, i.e. the order of token and prefix could be swapped without it being an issue, that would be preferable too